Modified the test script to fix the issue because of PR: https://gith…

rajneeshaec opened the pull request:
…ub.com/sonic-net/sonic-mgmt/pull/7816

### Description of PR
Modified the test script to fix the issue because of PR: https://github.com/sonic-net/sonic-mgmt/pull/7816


Summary:
Fixes # (issue)

### Type of change

- [ X] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012
- [Y ] 202205

### Approach
#### What is the motivation for this PR?
#### How did you do it?
This test case is failing for other vendors. That's why added marvell specific check to verify crm status


#### How did you verify/test it?
test case is passing on SONIC-PTF testbed
crm/test_crm.py::test_acl_entry[sonic-dut-None] PASSED

#### Any platform specific information?
Yes

mssonicbld commented:
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The pre-commit check is a mandatory check, please fix detected issues.

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/crm/test_crm.py:941:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:942:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:945:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:946:36: E128 continuation line under-indented for visual indent<br/><br/>check conditional mark sort..........................(no files to check)Skipped<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

Blueve commented:
/azp run

mssonicbld commented:
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The pre-commit check is a mandatory check, please fix detected issues.

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/crm/test_crm.py:941:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:942:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:945:36: E128 continuation line under-indented for visual indent<br/>tests/crm/test_crm.py:946:36: E128 continuation line under-indented for visual indent<br/><br/>check conditional mark sort..........................(no files to check)Skipped<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

Blueve commented:
@rajneeshaec can you fix the styling issue?

rajneeshaec commented:
> @rajneeshaec can you fix the styling issue?

Hi @Blueve 

  I have raised a separate PR and addressed the comments. Could you please check it:-
  https://github.com/sonic-net/sonic-mgmt/pull/8807
  
  Thanks
Rajneesh
